_rules/active-focus-shortcut-1e9941.md

```diff
@@ -0,0 +1,212 @@
+---
+id: 1e9941
+name: Keyboard shortcut is only active when component has focus

```

Comment:
The name should reflect the fact that the rule only checks printable characters shortcuts.